app/helpers/application_helper.rb

```diff
@@ -7,4 +7,8 @@ def user_image_tag(owner)
     email = owner.nil? ? nil : owner.email
     gravatar_image_tag(email)
   end
+
+  def any_minion?
+    !Minion.all.count.zero?
```

Comment:
That's something worth-investigating indeed. Then yeah, simply expose `Minion.any?` through this helper.